### PR TITLE
[SQLite] Implement simple property values query

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: internal/server/v1/propertyvalues/core.go [hunk 1]
```diff
@@ -32,8 +36,173 @@ import (
...
+	var resp map[string]map[string]map[string][]*pb.EntityInfo
+	var pg *pbv1.PaginationInfo
+	var err error
+	if store.BtGroup != nil {
+		resp, pg, err = fetchBT(ctx, store.BtGroup, nodes, properties, limit, token, direction)
+		if err != nil {
+			return nil, nil, err
+		}
+	}
+	// No pagination for sqlite query, so if there are pagination token, meaning
```
Comment (pradh) [inline internal/server/v1/propertyvalues/core.go:63]: nit: s/are/is/

**Review by pradh — APPROVED** [review]
Quick review

File: internal/server/v1/propertyvalues/core.go (hunk 1 above)
Comment (shifucun) [reply internal/server/v1/propertyvalues/core.go]: done

**Review by shifucun — COMMENTED** [review]
thanks for the review!

**Pull request merged**